x-pack/plugins/apm/server/lib/correlations/get_filters.ts

```diff
@@ -18,7 +17,6 @@ import {
 import { ProcessorEvent } from '../../../common/processor_event';
 
 export interface CorrelationsOptions {
-  setup: Setup;
```

Comment:
Not sure why `setup` was removed from here and added in other places.

Reply:
Wasn't used there,  I thought that was more convenient to extend the type where it's needed

Reply:
Okay, makes sense